pkg/systemd/journal.css

```diff
@@ -131,3 +131,8 @@
     white-space: nowrap;
     vertical-align: middle;
 }
+
+/* Make sure to not break log message lines in order to preserve information */
+#journal-entry .cockpit-info-table td {
+    white-space: normal;
```

Comment:
I think I only tried with strings with spaces in them. I'll test with super-long-strings and fix that too.